bucket/rustic.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.5.4",
+    "description": "fast, encrypted, and deduplicated backups powered by Rust",
+    "homepage": "https://rustic.cli.rs",
+    "license": "MIT|Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/rustic-rs/rustic/releases/download/v0.5.4/rustic-v0.5.4-x86_64-pc-windows-msvc.tar.gz",
+            "hash": "4a5074a2937dcf1b8bcc1bdeebdb9d10364062b00e0c6475b4e0d8859ebb49d2"
+        }
+    },
+    "env_set": {
+        "RUSTIC_HOME": "$dir"
```

Comment:
Hi @simonsan, where can I find documentation related to this environment variable?

Reply:
> `RUSTIC_HOME` is something special here though as that's something I added here, to be set, to work with for the future release. As usually `rustic` looks in `%ProgramData%\rustic\`, but for the scoop installation I think it might be nice to keep it closer to the user, as in `ProgramData` it would be only available to the user that initially created the files (afaiu).

 `%ProgramData%` is a standard location for system-wide application data. For user installations, it is `%AppData%`, which is present in the user home folder. This location is acceptable for Scoop applications, to store data. Even `~/.config` (which is the standard for *nix based systems) is acceptable. 

What I'm trying to say is we don't need to set this variable - because it takes away users' ability to set a location of their own.

Reply:
> it gives the possibility to set a location of their own to the users.

yeah, users can set a custom location with the environment variable, but `"RUSTIC_HOME": "$dir"` is defined in manifest,  `$env:RUSTIC_HOME` will be reset to `$dir` every update.

---

If you want to make the program more portable. `rustic_home` should be set in order and fallback to the first existing path:

0. `$env:RUSTIC_HOME`
1. `<program_root>\home`
2. `$env:APPDATA\.config\rustic` or `$env:USERPROFILE\.rustic`
3. `$env:APPDATA\rustic`
4. `$env:ProgramData\rustic`
...

note, this just a example or suggestion, or a feature request for your project.